### DISCUSSION TITLE
想要个性格的图标 ant好像只有单人的 没有组合的

### DISCUSSION THREAD

**Discussion by 2010363329** [ant-design/ant-design#46647] [Q&A]
<img width="30" alt="image" src="https://github.com/ant-design/ant-design/assets/76477581/027fe49c-1c51-4b07-b10b-573dc62e6426">
像这样一样 字节的库也没有 lucide也没有（人家还把我的qa删了 呜呜～）

**Comment by Wxh16144** [comment]
看看这里的回答： #46280

**Reply by 2010363329** [reply]
看了 lucide里面没找到 发了个qa给我直接关闭了 iconpark也没有 不过你咋还记得我直接发的 哈哈 主要是觉得ant好用呀

**Reply by Wxh16144** [reply]
在 [这里](https://icones.js.org/collection/all?s=gender) 找一个符合你项目的协议的用

**Comment by yoyo837** [comment]
怎么可能任意想到的都提供了，不满足的自己画或者团队设计师画。

**Reply by yoyo837** [reply]
iconfont等图标库里好像也能用别人上传的
